docs/development/documentation_guidelines.rst

```diff
@@ -29,7 +29,10 @@ or ::
     cd docs
     make html
 
-.. note :: If you're working on a fresh local copy of the TARDIS repository, you might need to do ``python setup.py develop`` before executing these commands.
+.. note :: 
+
+    - If you're working on a fresh local copy of the TARDIS repository, you might need to do ``python setup.py develop`` before executing these commands.
+    - Prepend ``DISABLE_NBSPHINX=1`` to the build command to disable notebook rendering (fast mode).
```

Comment:
I'm not entirely sure what you mean by prepend here. Would it be `DISABLE_NBSPHINX=1 python setup.py develop` as the whole command?

Reply:
yes, It's basically an env variable. 

Also you can use `export DISABLE_NBSPHINX=1` and then run the build command.

Reply:
maybe line 75 on the disable nbsphinx to say `never`

Reply:
done!